airbyte-workers/src/test-integration/java/io/airbyte/workers/process/KubePodProcessIntegrationTest.java

```diff
@@ -121,6 +119,31 @@ public void teardown() throws Exception {
     server.stop();
   }
 
+  @RetryingTest(3)
+  public void testInitKubePortManagerSingletonTwice() throws Exception {
```

Comment:
@davinchia what I really add test here.

Others code change is coding-style formatter by google-style which recommends (https://github.com/airbytehq/airbyte/blob/master/docs/contributing-to-airbyte/code-style.md)